addons/crm/data/access_request_mail_template_data.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <!-- TDE note: this should not be a mail template -->
+        <record id="mail_template_crm_access_request" model="mail.template">
```

Comment:
In fact, can't we still use the default one `mail_template_base_install_request`, it seems to only change some wording so I would say it's not worth it, especially when it adds python code just to get the other template